File: Projects/Ex54MaterialTest/app/src/main/java/com/c282804/ex54materialtest/IntroActivity.java
package com.c282804.ex54materialtest;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.os.Message;

public class IntroActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_intro);

        //4초후에 MainActivity를 자동 실행하기
        handler.sendEmptyMessageDelayed(0,4000);
    }
    Handler handler = new Handler(){
        @Override
        public void handleMessage(@NonNull Message msg) {
            Intent intent =new Intent(IntroActivity.this,MainActivity.class);
            startActivity(intent);

            finish();


        }
    };

}

File: Projects/Ex91KotlinRecyclerViewApp/app/src/main/java/com/c282804/ex91kotlinrecyclerviewapp/IntroActivity.kt
package com.c282804.ex91kotlinrecyclerviewapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_intro.*

class IntroActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_intro)

        //id가 btn인 버튼에 클릭리스너 추가하기
        btn.setOnClickListener(object:View.OnClickListener{
            override fun onClick(v: View?) {
                //MainActivity 실행
                var intent= Intent(this@IntroActivity,MainActivity::class.java)
                startActivity(intent)
            }
        })
    }

    fun clickExit(view: View) {
        finish()
    }
}
